src/content/overview/observability/alerting/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Alerting with the Observability Platform"
+linkTitle: Alerting
+description: Documentation on the observability-platform alerting concept and architecture deployed and maintained by Giant Swarm.
+weight: 80
+menu:
+  principal:
+    identifier: overview-observability-alerting
+    parent: overview-observability
+user_questions:
+  - How is alerting managed by the observability platform?
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+last_review_date: 2025-04-24
+---
+
+Alerting is an important concept of any observability solution and it's thus only natural that it is available as part of the Giant Swarm Observability Platform. For more details and information on alerting please visit the [official documentation page](https://grafana.com/docs/grafana/latest/alerting/).
+
+Alerting is usually divided into two distinct concepts: the alerting pipeline (how to send alerts, to whom and what to send) and alerting/recording rules (what to alert on). This documentation will cover how those two topics work in the Observability Platform.
```

Comment:
You coud add this to the "How alerting works" section. Directly above the multi-tenancy reference. I think it makes sense having both together in that subchapter.